featurebyte/service/credential.py

```diff
@@ -82,40 +79,52 @@ def construct_list_query_filter(
             output["user_id"] = self.user.id
         return output
 
-    async def _validate_credential(self, credential: CredentialModel) -> None:
+    # This is a temporary method
+    # This is hardcoded to use user_id to find the credential
+    async def find(
+        self, user_id: Optional[ObjectId], feature_store_name: str
```

Comment:
Can we change this to use `feature_store_id` instead of `feature_store_name`?

It seems that we always have a feature store id when making this call so we can skip redundant mongodb call to fetch the id